app/services/symbolService.js

```diff
@@ -96,8 +96,68 @@ class SymbolService {
     if (params.on_time !== undefined) processedSymbol.on_time = params.on_time;
     if (params.status !== undefined) processedSymbol.status = params.status;
 
-    const newExchangeSymbol = await new DataExchangeSymbolModel(processedSymbol).save();
-    return { newExchangeSymbol };
+    const newDataExchangeSymbol = await new DataExchangeSymbolModel(processedSymbol).save();
+    return { newDataExchangeSymbol };
+  }
```

Comment:
What is the purpose of using {}?

Reply:
i ain't write that. Ying did. Not sure why it appeared here

Reply:
You need to confirm that every line of code you submit is written by yourself. Please do not modify someone else's code or submit unknown code without their consent.

Reply:
I agree with you. And i always keep that in mind.